main.go

```diff
@@ -330,6 +347,13 @@ func transformCompile(args []string) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if envGarbleDebugDir != "" {
+		deferred = append(deferred, func() error {
+			return copyPackage(pkgPath, tempDir, envGarbleDebugDir)
+		})
```

Comment:
instead of this, what I would do is make it so that we write to both locations as we are transforming the files. this is significantly simpler, and we don't need to re-open the temporary files.

Reply:
ok